packages/web-api-scan-runner/src/report-generator/axe-result-to-html-converter.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AxeReportParameters, ReporterFactory } from 'accessibility-insights-report';
+import { AxeResults } from 'axe-core';
+import { inject, injectable } from 'inversify';
+import { ReportFormat } from 'storage-documents';
+import { iocTypeNames } from '../ioc-types';
+import { AxeResultConverter, ReportGenerationParams } from './axe-result-converter';
+
+@injectable()
+export class AxeResultToHtmlConverter implements AxeResultConverter {
+    public readonly reportType: ReportFormat = 'html';
+
+    constructor(@inject(iocTypeNames.ReporterFactory) private readonly reporterFactoryFunc: ReporterFactory) {}
+
+    public convert(results: AxeResults, params: ReportGenerationParams): string {
+        const reporter = this.reporterFactoryFunc();
+
+        const htmlReportParams: AxeReportParameters = {
+            results: results,
+            description: 'Automated report',
+            serviceName: 'Accessibility Insights Service',
```

Comment:
NIT: these strings could be put at a central place like what we are doing in web repo: https://github.com/microsoft/accessibility-insights-web/blob/master/src/content/strings/application.ts